### DISCUSSION TITLE
Do you have plan build a native version of Scoop?

### DISCUSSION THREAD

**Discussion by silenceleaf** [ScoopInstaller/Scoop#5880] [Ideas]
## Feature Request

Scoop is written by powershell. As a package manager, do you have plan build a binary version? (a standalone exe)

If no, do you mind community build one?

**Comment by rashil2000** [comment]
There are no plans to build a standalone binary. 

Even a standalone binary would require helper shell scripts, inevitably, in order to modify the env vars in the current session(s).
